test/gas-analytics/SpokePool.RelayerRefundRootExecution.ts

```diff
@@ -244,5 +245,32 @@ describe("Gas Analytics: SpokePool Relayer Refund Root Execution", function () {
       const receipt = await txn.wait();
       console.log(`executeRelayerRefundRoot-gasUsed: ${receipt.gasUsed}`);
     });
+    it(`Stress Test: 1 leaf contains ${STRESS_TEST_REFUND_COUNT} refunds with amount > 0`, async function () {
+      // This test should inform the limit # refunds that we would allow a RelayerRefundLeaf to contain to avoid
```

Comment:
Do we have any information on limits on most L2s? If you don't know, that's totally fine, we should just add a TODO to figure out a good limit that's below all L2s that we're deploying to.